source/platforms/databricks.md

```diff
@@ -44,15 +44,15 @@ At the time of writing the `databricksruntime/gpu-pytorch:cuda11.8` image does n
 
 ````
 
-At the top of your notebook run any of the following `pip` install commands to install your preferred RAPIDS libraries.
+At the top of your notebook run any of the following pip install commands to install your preferred RAPIDS libraries.
 
 ```text
 !pip install cudf-cu11 dask-cudf-cu11 --extra-index-url=https://pypi.nvidia.com
 !pip install cuml-cu11 --extra-index-url=https://pypi.nvidia.com
 !pip install cugraph-cu11 --extra-index-url=https://pypi.nvidia.com
 ```
 
-### Test Rapids
+### Test RAPIDS
 
 ```python
 import cudf
```

Comment:
When I do this I see the following exception.

```pytb
ValueError: pyarrow.lib.IpcWriteOptions size changed, may indicate binary incompatibility. Expected 88 from C header, got 72 from PyObject
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
File <command-393379380291571>, line 1
----> 1 import cudf
      3 gdf = cudf.DataFrame({"a":[1,2,3],"b":[4,5,6]})
      4 gdf

File /local_disk0/.ephemeral_nfs/envs/pythonEnv-dcdef616-1f9f-4d00-8058-b5c9cd579d96/lib/python3.10/site-packages/cudf/__init__.py:19
     16 from rmm.allocators.cupy import rmm_cupy_allocator
     17 from rmm.allocators.numba import RMMNumbaManager
---> 19 from cudf import api, core, datasets, testing
     20 from cudf.api.extensions import (
     21     register_dataframe_accessor,
     22     register_index_accessor,
     23     register_series_accessor,
     24 )
     25 from cudf.api.types import dtype

File /local_disk0/.ephemeral_nfs/envs/pythonEnv-dcdef616-1f9f-4d00-8058-b5c9cd579d96/lib/python3.10/site-packages/cudf/datasets.py:7
      4 import pandas as pd
      6 import cudf
----> 7 from cudf._lib.transform import bools_to_mask
      8 from cudf.core.column_accessor import ColumnAccessor
     10 __all__ = ["timeseries", "randomdata"]

File /local_disk0/.ephemeral_nfs/envs/pythonEnv-dcdef616-1f9f-4d00-8058-b5c9cd579d96/lib/python3.10/site-packages/cudf/_lib/__init__.py:4
      1 # Copyright (c) 2020-2023, NVIDIA CORPORATION.
      2 import numpy as np
----> 4 from . import (
      5     avro,
      6     binaryop,
      7     concat,
      8     copying,
      9     csv,
     10     datetime,
     11     expressions,
     12     filling,
     13     groupby,
     14     hash,
     15     interop,
     16     join,
     17     json,
     18     labeling,
     19     merge,
     20     null_mask,
     21     nvtext,
     22     orc,
     23     parquet,
     24     partitioning,
     25     pylibcudf,
     26     quantiles,
     27     reduce,
     28     replace,
     29     reshape,
     30     rolling,
     31     round,
     32     search,
     33     sort,
     34     stream_compaction,
     35     string_casting,
     36     strings,
     37     strings_udf,
     38     text,
     39     timezone,
     40     transpose,
     41     unary,
     42 )
     44 MAX_COLUMN_SIZE = np.iinfo(np.int32).max
     45 MAX_COLUMN_SIZE_STR = "INT32_MAX"

File avro.pyx:1, in init cudf._lib.avro()

File column.pyx:1, in init cudf._lib.column()

File scalar.pyx:25, in init cudf._lib.scalar()

File interop.pyx:1, in init cudf._lib.interop()

ValueError: pyarrow.lib.IpcWriteOptions size changed, may indicate binary incompatibility. Expected 88 from C header, got 72 from PyObject
```

Reply:
Seeing the same error  testing `cudf-cu11` on: 
- `databricksruntime/gpu-tensorflow:cuda11.8` and 
- `databricksruntime/gpu-pytorch:cuda11.8`


The [error ](https://github.com/huggingface/datasets/issues/5923)looks related so i tried downgrade `pyarrow==11.0.0` and upgrade to `pyarrow==14.0.1`:

- `ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
cudf-cu11 23.10.1 requires pyarrow==12.*, but you have pyarrow 14.0.1 which is incompatible.`

Reply:
I also tried testing with `12.2 LTS ML GPU` and `13.3 LTS ML GPU` runtimes to see if we could bypass the docker containers but fails in both cases: 
```
ImportError: 
================================================================
Failed to import CuPy.

If you installed CuPy via wheels (cupy-cudaXXX or cupy-rocm-X-X), make sure that the package matches with the version of CUDA or ROCm installed.

On Linux, you may need to set LD_LIBRARY_PATH environment variable depending on how you installed CUDA/ROCm.
On Windows, try setting CUDA_PATH environment variable.

Check the Installation Guide for details:
  https://docs.cupy.dev/en/latest/install.html

Original error:
  ImportError: libcudart.so.12: cannot open shared object file: No such file or directory

```

^^ I tried to manually install cuda-toolkit 12  following [instructions](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu&target_version=22.04&target_type=deb_network) but it  didn't work

Reply:
Awesome ok I tested things with `12.2 LTS Runtime` and the `databricksruntime/gpu-tensorflow:cuda11.8` container.

Looking at `nvidia-smi` it shows CUDA 12.2 so I upgraded CUDA toolkit with

```bash
!cd /etc/apt/sources.list.d && \
    mv cuda-ubuntu2204-x86_64.list.disabled cuda-ubuntu2204-x86_64.list && \
    apt-get update && apt-get --no-install-recommends -y install cuda-toolkit-12.2 && \
    mv cuda-ubuntu2204-x86_64.list cuda-ubuntu2204-x86_64.list.disabled
```

Then I installed `cudf-cu12` and `dask-cudf-cu12`.

```python
!pip config set global.extra-index-url https://pypi.nvidia.com
!pip install cudf-cu12 dask-cudf-cu12
```

And things worked as expected!

```
>>> import cudf
>>> gdf = cudf.DataFrame({"a":[1,2,3],"b":[4,5,6]})
>>> gdf
	a	b
0	1	4
1	2	5
2	3	6
```